ros_ign_bridge/CMakeLists.txt

```diff
@@ -105,6 +105,10 @@ foreach(bridge ${bridge_executables})
   target_link_libraries(${bridge}
     ignition-msgs${IGN_MSGS_VER}::core
     ignition-transport${IGN_TRANSPORT_VER}::core
+    # ros_ign_interfaces libraries
+    -lros_ign_interfaces__rosidl_typesupport_cpp
+    -lros_ign_interfaces__rosidl_typesupport_fastrtps_cpp
+    -lros_ign_interfaces__rosidl_typesupport_introspection_cpp
```

Comment:
```suggestion
```

Reply:
You should include `ros_ign_interfaces`  in `ament_target_dependencies` like another ros2 package